src/tag_serial_driver.cpp

```diff
@@ -92,29 +92,42 @@ int serial_setup(const char * device)
   return fd;
 }
 
-void receive_ver_req(const std_msgs::msg::Int32::ConstSharedPtr msg)
+void receive_ver_req([[maybe_unused]] const std_msgs::msg::Int32::ConstSharedPtr msg)
 {
   char ver_req[] = "$TSC,VER*29\x0d\x0a";
-  write(fd, ver_req, sizeof(ver_req));
-  RCLCPP_INFO(rclcpp::get_logger("tag_serial_driver"), "Send Version Request:%s", ver_req);
+  int ver_req_data = write(fd, ver_req, sizeof(ver_req));
+  if (ver_req_data >= 0) {
+    RCLCPP_INFO(rclcpp::get_logger("tag_serial_driver"), "Send Version Request:%s", ver_req);
+  } else {
+    RCLCPP_ERROR(rclcpp::get_logger("tag_serial_driver"), "Send Version Request: error");
```

Comment:
How about this? Probably we want to know what message we couldn't send.

```suggestion
    RCLCPP_ERROR(rclcpp::get_logger("tag_serial_driver"), "Send Version Request%s: error", ver_req);
```

Reply:
That sounds good.

Reply:
fixed in https://github.com/tier4/tamagawa_imu_driver/pull/5/commits/454f5336cf795283b21b0b0bbd367d106fd365f6